tests/acceptance/TestHelpers/AuthAppHelper.php

```diff
@@ -81,6 +81,32 @@ public static function createAppAuthToken(
 		);
 	}
 
+	/**
+	 * @param string $baseUrl
+	 * @param string $user
+	 * @param string $password
+	 * @param string $expiration
+	 * @param string $impersonatorUser
+	 *
+	 * @return ResponseInterface
+	 */
+	public static function createAuthAppTokenViaImpersonationAPI(
```

Comment:
do we need separate method? we can pass the array of query params to the above method (createAppAuthToken), build the params and pass to the request